conda-env.yml

```diff
@@ -6,8 +6,13 @@ dependencies:
   - cython
   - ddt
   - gitdb
+  - gitpython
+  - h5py
+  - hdf5plugin
+  - lz4
+  - matplotlib
+  - numpy
```

Comment:
doesn't some of these cause conflicts with packages already in requirements.txt?

Reply:
the overlap allows for a pip-only-installation, it's not super clean but it doesn't cause conflicts that would crash anything

Reply:
Ok, I was wondering because I have add some issues before where same package defined in requirements.txt and within docs/requirements.txt which also included `-r requirements.txt` line caused double entry conflict.
If conda handles it correctly, that's fine.